packages/cli/src/commands/init/templateName.js

```diff
@@ -76,10 +76,20 @@ async function tryTemplateShorthand(templateName: string) {
     return templateName;
   }
   try {
-    const reactNativeTemplatePackage = `react-native-template-${templateName}`;
-    const response = await fetch(
-      `https://registry.yarnpkg.com/${reactNativeTemplatePackage}/latest`,
-    );
+    const nameTagArray = templateName.split('@');
+    const name = nameTagArray[0];
+    const tag = nameTagArray[1];
```

Comment:
let's assign a default value here, so we can avoid branching:
```suggestion
    const tag = nameTagArray[1] || 'latest';
```